Form/Field/EnvironmentPickerType.php

```diff
@@ -77,7 +68,7 @@ public function configureOptions(OptionsResolver $resolver)
                     'data-live-search' => false
             ],
             'choice_attr' => function ($category, $key, $index) {
-                /** @var \EMS\CoreBundle\Form\DataField\DataFieldType $dataFieldType */
+                /** @var Environment $dataFieldType */
                 $dataFieldType = $this->choices[$index];
```

Comment:
We should also rename the var

Reply:
I agree with renaming, and we should add an "instanceOf" check to prevent errors.